CONTRIBUTING.md

```diff
@@ -85,45 +99,23 @@ Ensure you have the following installed:
 4. Update documentation if your changes affect the provider's behavior or add new features.
 
 ## Testing
+Unit tests can be run with `make test`.
 
-1. Run unit tests:
-   ```
-   make test
-   ```
+### Acceptance tests
+Acceptance integration tests use a Terraform CLI binary to run real Terraform commands against the Astro API. The goal is to approximate using the provider with Terraform in production as closely as possible.
 
-2. Run acceptance tests (these will create real resources in your Astro account):
-   ```
-   make testacc
-   ```
-   Note: Ensure all required environment variables are set as described in `internal/provider/provider_test_utils.go`.
+Using the terraform-plugin-testing framework, each `resource.Test` runs an acceptance test on a resource.
+- `ProtoV6ProviderFactories`: map of the provider factories that the test suite will use to create the provider - just has the `astronomer` provider
+- `PreCheck`: a function that runs before the test suite starts to check that all the required environment variables are set
+- `Steps`: a list of `terraform apply` sequences that the test suite will run. Each step is a `resource.TestStep` that contains a `Config` and `Check` function.
+  - `Config`: the Terraform configuration that the test will run (ie. the `.tf` file)
+  - `Check`: function that will verify the state of the resources after the `terraform apply` command has run.
 
-3. Test your changes manually using the `main.tf` file you created earlier:
-   ```
-   terraform init
-   terraform plan
-   terraform apply
-   ```
```

Comment:
i would undo all the changes to delete the testing info, i think it would be better to combine the two and fix the wording instead